sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/statsbeat/StatsbeatModule.java

```diff
@@ -80,11 +82,13 @@ public void start(
         updateConnectionString(connectionString.get());
         updateInstrumentationKey(instrumentationKey.get());
 
-        scheduledExecutor.scheduleWithFixedDelay(
-            new StatsbeatSender(networkStatsbeat, telemetryItemExporter),
-            shortIntervalSeconds,
-            shortIntervalSeconds,
-            TimeUnit.SECONDS);
+        if (RpAttachType.getRpAttachType() != RpAttachType.MANUAL) {
```

Comment:
`RpAttachType.STANDALONE` (standolone exporter) would be more explicit for me, rather than `RpAttachType.MANUAL`

Reply:
again it's based on spec.